packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -516,4 +518,51 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /internal/patient/
+ *
+ * WIP
+ *
+ */
+router.get(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const dob = getFrom("query").orFail("dob", req);
+    const genderAtBirth = genderAtBirthSchema.parse(getFrom("query").orFail("genderAtBirth", req));
+    const firstNameInitial = getFrom("query").optional("firstNameInitial", req);
+    const lastNameInitial = getFrom("query").optional("lastNameInitial", req);
+    const foundPatients = await patientLoader.findBySimilarity({
+      cxId,
+      data: {
+        dob,
+        genderAtBirth,
+        firstNameInitial,
+        lastNameInitial,
+      },
+    });
+    // TODO check if we're not returning Sequelize's Model data here; even thought the shape is Patient, the underlying object is PatientModel
+    // If we are, we should convert it to a DTO. Here, on `GET /internal/patient/:id`, and on `GET /internal/mpi/patient`
```

Comment:
I added `dtoFromModel` calls in line with what is done in the `patient.ts` and `internal-patient.ts` routes

Reply:
That sounds good, then we also need to update the caller... it expects the domain shape, not the DTO.

Ideally, we would have convert to a schema shape and use that same schema to validate on the caller side, but it might be at this time. Maybe keep the `toDTO` and build a `fromDTO`?

Alternatively, we could just keep the domain shape for output/input and use `.get({ plain: true })` if its an instance of `Model`

Reply:
added fromDTO